hack/cri-containerd.toml

```diff
@@ -0,0 +1,12 @@
+version = 2
+
+disabled_plugins = []
```

Comment:
all mentioned plugins are disabled in the containerd build, no need to disable them once again